angr/analyses/decompiler/structured_codegen.py

```diff
@@ -404,17 +404,17 @@ def c_repr_chunks(self, indent=0):
 
         indent_str = self.indent_str(indent=indent)
 
-        yield indent_str, self
+        yield indent_str, None
```

Comment:
I feel I've seen these changes before... did you rebase on master?

Reply:
just rebased